docs/resources/Invite.md

```diff
@@ -2,15 +2,18 @@
 
 ### Invite Object
 
-Represents a code that when used, adds a user to a guild or group DM channel.
+Represents a code that when used, adds a user to a guild or group DM channel or adds a user to the inviter's friends list.
+
+> warn
+> All fields except `code` and `inviter` will be null or absent on friend invites.
```

Comment:
this isn't true (it can have expires_at), and I don't think this warning block is necessary